.github/workflows/on_pr.yml

```
@@ -56,7 +63,7 @@ jobs:
   #                 still finishing (e.g. Mergify CI auto-retry). Treat as
   #                 non-success — do not whitelist only failure/cancelled.
   all:
-    needs: [ main_workflow ]
+    needs: [ main_workflow, upload_metrics_history ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)